### PR TITLE
CC-21132 AC Documentation Support configurable products in the shopping list

### PR DESCRIPTION
**Pull request by VadymSachenko**
## PR Description
Jira ticket: https://spryker.atlassian.net/browse/CC-21132
Epic: https://spryker.atlassian.net/browse/CC-9276

TBD

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

**Review by DmitryLymarenko — DISMISSED** [review]

**Review by jeremyfourna — DISMISSED** [review]
Ok for me

File: docs/pbc/all/shopping-list-and-wishlist/manage-via-glue-api/manage-shopping-list-items-via-glue-api.md [hunk 1]
```diff
@@ -69,16 +88,85 @@ To add items to a shopping list, send the request:
...
+</details>
 
 | ATTRIBUTE | TYPE | REQUIRED | DESCRIPTION |
 | --- | --- | --- | --- |
-| quantity | Integer | ✓ | Quantity of the product to add. |
-| sku | String | ✓ | SKU of the product to add. Only [concrete products](/docs/scos/user/features/{{site.version}}/product-feature-overview/product-feature-overview.html) are allowed. |
+| quantity | Integer | &check; | Quantity of the product to add. |
+| sku | String | &check; | SKU of the product to add. Only [concrete products](/docs/scos/user/features/{{site.version}}/product-feature-overview/product-feature-overview.html) and [configurable products](/docs/scos/user/features/{{site.version}}/configurable-product-feature-overview.html) are allowed. |
+| productOfferReference | String | &check; | Unique identifier of the product offer. |
+|merchantReference| String | &check; | Unique identifier of the merchant. |
```
Comment (lenadoc) [inline docs/pbc/all/shopping-list-and-wishlist/manage-via-glue-api/manage-shopping-list-items-via-glue-api.md:154]: we do not have the merchantReference in the response sample. Please check if it should be added to the response sample or removed from the attributes here

File: docs/pbc/all/shopping-list-and-wishlist/manage-via-glue-api/manage-shopping-list-items-via-glue-api.md [hunk 2]
```diff
@@ -218,15 +397,79 @@ To change item quantity in a shopping list, send the request:
...
+
 | ATTRIBUTE | TYPE | REQUIRED | DESCRIPTION |
-| --- | --- | --- |--- |
-| sku | String | ✓ | SKU of the product you want to change the quantity of. Only [concrete products](/docs/scos/user/features/{{site.version}}/product-feature-overview/product-feature-overview.html) are allowed. |
-| quantity | Integer | ✓ | New quantity of the product. |
+| --- | --- | --- | --- |
+| quantity | Integer | &check; | New quantity of the product. |
+| sku | String | &check; | SKU of the product you want to change the quantity of. Only [concrete products](/docs/scos/user/features/{{site.version}}/product-feature-overview/product-feature-overview.html) and [configurable products](/docs/scos/user/features/{{site.version}}/configurable-product-feature-overview.html) are allowed. |
+| productOfferReference | String | &check; | Unique identifier of the product offer. |
+|merchantReference| String | &check; | Unique identifier of the merchant. |
```
Comment (lenadoc) [inline docs/pbc/all/shopping-list-and-wishlist/manage-via-glue-api/manage-shopping-list-items-via-glue-api.md:461]: same here as above for the merchantReference

File: docs/pbc/all/shopping-list-and-wishlist/manage-via-glue-api/manage-shopping-list-items-via-glue-api.md [hunk 3]
```diff
@@ -351,8 +668,8 @@ If the item is removed successfully, the endpoint returns the `204 No Content` s
 | 400 | Provided access token is not an [access token of a company user](/docs/scos/dev/glue-api-guides/{{site.version}}/managing-b2b-account/authenticating-as-a-company-user.html). |
 | 901 | Shop list name or item name is not specified or too long.<br>**OR** <br> Item quantity is not specified or too large.|
```
Comment (lenadoc) [inline docs/pbc/all/shopping-list-and-wishlist/manage-via-glue-api/manage-shopping-list-items-via-glue-api.md:669]:
```suggestion
| 901 | Shop list name or item name is not specified or too long.<br>OR<br> Item quantity is not specified or too large.|
```

File: docs/pbc/all/shopping-list-and-wishlist/manage-via-glue-api/manage-shopping-list-items-via-glue-api.md [hunk 4]
```diff
@@ -69,16 +88,85 @@ To add items to a shopping list, send the request:
...
 | ATTRIBUTE | TYPE | REQUIRED | DESCRIPTION |
 | --- | --- | --- | --- |
-| quantity | Integer | ✓ | Quantity of the product to add. |
-| sku | String | ✓ | SKU of the product to add. Only [concrete products](/docs/scos/user/features/{{site.version}}/product-feature-overview/product-feature-overview.html) are allowed. |
+| quantity | Integer | &check; | Quantity of the product to add. |
+| sku | String | &check; | SKU of the product to add. Only [concrete products](/docs/scos/user/features/{{site.version}}/product-feature-overview/product-feature-overview.html) and [configurable products](/docs/scos/user/features/{{site.version}}/configurable-product-feature-overview.html) are allowed. |
+| productOfferReference | String | &check; | Unique identifier of the product offer. |
+|merchantReference| String | &check; | Unique identifier of the merchant. |
+| productConfigurationInstance.displayData  | Array  |&check; | Array of variables that are proposed for a Storefront user to set up in the configurator.  |
+| productConfigurationInstance.configuration  | Array  | &check; | Default configurable product configuration.  |
```
Comment (lenadoc) [inline docs/pbc/all/shopping-list-and-wishlist/manage-via-glue-api/manage-shopping-list-items-via-glue-api.md:156]:
```suggestion
| productConfigurationInstance.configuration  | Array  | &check; | Default configuration of the configurable product.  |
```
As Default configurable product configuration might be misunderstood as configuration of a default configurable product"

File: docs/pbc/all/shopping-list-and-wishlist/manage-via-glue-api/manage-shopping-list-items-via-glue-api.md [hunk 5]
```diff
@@ -168,15 +256,85 @@ To add items to a shopping list, send the request:
...
 
 | ATTRIBUTE | TYPE | DESCRIPTION |
 | --- | --- | --- |
-| cell | cell | cell |
+| productOfferReference | String | Unique identifier of the product offer. |
+|merchantReference| String | Unique identifier of the merchant. |
 | quantity | Integer | Quantity of the product. |
 | sku | String | Product SKU. |
+| productConfigurationInstance.displayData  | Array  | Array of variables that are proposed for a Storefront user to set up in the configurator.  |
+| productConfigurationInstance.configuration  | Array  | Default configurable product configuration.  |
```
Comment (lenadoc) [inline docs/pbc/all/shopping-list-and-wishlist/manage-via-glue-api/manage-shopping-list-items-via-glue-api.md:329]:
```suggestion
| productConfigurationInstance.configuration  | Array  | Default configuration of the configurable product.  |
```

File: docs/pbc/all/shopping-list-and-wishlist/manage-via-glue-api/manage-shopping-list-items-via-glue-api.md [hunk 6]
```diff
@@ -218,15 +397,79 @@ To change item quantity in a shopping list, send the request:
...
-| --- | --- | --- |--- |
-| sku | String | ✓ | SKU of the product you want to change the quantity of. Only [concrete products](/docs/scos/user/features/{{site.version}}/product-feature-overview/product-feature-overview.html) are allowed. |
-| quantity | Integer | ✓ | New quantity of the product. |
+| --- | --- | --- | --- |
+| quantity | Integer | &check; | New quantity of the product. |
+| sku | String | &check; | SKU of the product you want to change the quantity of. Only [concrete products](/docs/scos/user/features/{{site.version}}/product-feature-overview/product-feature-overview.html) and [configurable products](/docs/scos/user/features/{{site.version}}/configurable-product-feature-overview.html) are allowed. |
+| productOfferReference | String | &check; | Unique identifier of the product offer. |
+|merchantReference| String | &check; | Unique identifier of the merchant. |
+| productConfigurationInstance.displayData  | Array  |&check; | Array of variables that are proposed for a Storefront user to set up in the configurator.  |
+| productConfigurationInstance.configuration  | Array  | &check; | Default configurable product configuration.  |
```
Comment (lenadoc) [inline docs/pbc/all/shopping-list-and-wishlist/manage-via-glue-api/manage-shopping-list-items-via-glue-api.md:463]:
```suggestion
| productConfigurationInstance.configuration  | Array  | &check; | Default configuration of the configurable product.  |
```

File: docs/pbc/all/shopping-list-and-wishlist/manage-via-glue-api/manage-shopping-list-items-via-glue-api.md [hunk 7]
```diff
@@ -310,10 +553,84 @@ To change item quantity in a shopping list, send the request:
...
+</details>
+
+| ATTRIBUTE | TYPE | REQUIRED | DESCRIPTION |
+| --- | --- | --- | --- |
+| quantity | Integer | &check; | Updated quantity of the product. |
+| sku | String | &check; | SKU of the product whose quantity has been updated. Only [concrete products](/docs/scos/user/features/{{site.version}}/product-feature-overview/product-feature-overview.html) and [configurable products](/docs/scos/user/features/{{site.version}}/configurable-product-feature-overview.html) are allowed. |
+| productOfferReference | String | &check; | Unique identifier of the product offer. |
+|merchantReference| String | &check; | Unique identifier of the merchant. |
+| productConfigurationInstance.displayData  | Array  |&check; | Array of variables that are proposed for a Storefront user to set up in the configurator.  |
+| productConfigurationInstance.configuration  | Array  | &check; | Default configurable product configuration.  |
```
Comment (lenadoc) [inline docs/pbc/all/shopping-list-and-wishlist/manage-via-glue-api/manage-shopping-list-items-via-glue-api.md:625]:
```suggestion
| productConfigurationInstance.configuration  | Array  | &check; | Default configuration of the configurable product.  |
```

**Review by lenadoc — CHANGES_REQUESTED** [review]

File: docs/pbc/all/shopping-list-and-wishlist/manage-via-glue-api/manage-shopping-list-items-via-glue-api.md (hunk 1 above)
Comment (VadymSachenko) [reply docs/pbc/all/shopping-list-and-wishlist/manage-via-glue-api/manage-shopping-list-items-via-glue-api.md:154]: We don't we indeed. Will remove it.

**Review by lenadoc — APPROVED** [review]

**Pull request merged**